nautobot/extras/templates/extras/customfield.html

```diff
@@ -34,10 +33,6 @@
                         <td>Weight</td>
                         <td><span>{{ object.weight }}</span></td>
                     </tr>
-                    <tr>
-                        <td>Label</td>
```

Comment:
Looks like `slug` isn't currently on this page? Given its relevance for custom fields it might be good to add even if it's a bit redundant with the `advanced` tab.